source/PluginDev/Assets/GooglePlayGames/BasicApi/Events/IEvent.cs

```diff
@@ -16,7 +16,7 @@
 
 namespace GooglePlayGames.BasicApi.Events
 {
-
+    using GooglePlayGames.Native.PInvoke;
```

Comment:
>  using GooglePlayGames.Native.PInvoke

Can't see usage of this in this file.

Reply:
`EventManager` is from this namespace. If we remove this `using`, we have to specify full name for `EventManager` in `cref="EventManager"` annotation of `IEvent`.

Reply:
I've removed `using` and specified full name in annotation.